db/migrate/20200218213414_add_start_and_end_dates_to_plans.rb

```diff
@@ -0,0 +1,7 @@
+class AddStartAndEndDatesToPlans < ActiveRecord::Migration
+  def change
+    add_column :plans, :grant_id, :integer, index: true
```

Comment:
If you end up changing grant to an association, this may need to change to a reference to enforce the foreign_key 
`add_reference :plans, :grant`